sdk/resources/azure-mgmt-resourcegraph/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 2.2.0 (2020-10-16)
```

Comment:
According the result of last meeting, version number of python sdk for track should begin with "track1 version + 5". So the version number should be 7.0.0. (remember change _version.py as well)